packages/grafana-ui/src/components/InfoBox/InfoBox.story.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { number } from '@storybook/addon-knobs';
+import { InfoBox } from './InfoBox';
+
+const InfoBoxStories = storiesOf('General/InfoBox', module);
```

Comment:
this is the old style of defining stories: please use https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/components/ThemeColors/ThemeColors.story.tsx#L4